### ISSUE TITLE
Config location / XDG

### ISSUE THREAD

**Issue by SamHH** [osa1/tiny#152]
Would it be possible to also look for the config file in the directory specified by `$XDG_CONFIG_HOME`, or the default fallback as defined [here](https://standards.freedesktop.org/basedir-spec/latest/ar01s03.html)? So it'd be for example: `~/.config/tiny/config.yml`

Also, as an aside, just wanted to say following the three issues I've filed that I really love this IRC client, so thanks :slightly_smiling_face:

**Comment by osa1** [comment]
:+1: Following the standards is a good idea, we should implement this.

There's even a crate (xdg) for this, but it's probably not necessary for something this simple.

> Also, as an aside, just wanted to say following the three issues I've filed that I really love this IRC client, so thanks

I'm glad you liked it!

**Comment by osa1** [comment]
This should be fixed. @SamHH please let me know if you have any problems.

**Comment by SamHH** [comment]
Thanks!

Do you know if the `tiny-irc-client-git` package needs updating in the AUR? I'm a bit unclear on how `*-git` packages work.

**Comment by osa1** [comment]
I'm not an Arch user myself, but looking at PKGBUILD, it seems like it's building from the source with the right compiler and dependencies, so I think it should work. @wrycode may say more about this.

**Comment by wrycode** [comment]
@SamHH @osa1 my -git package should work fine (I tested it after the recent big refactor a few weeks ago), but ping me if you have any problems with it.

Some other info:
There is also the [non *-git AUR package](https://aur.archlinux.org/packages/tiny-irc-client/). I checked it out at one point and it was legit when I looked at it. *-git packages just build programs from the latest commit, while non *-git packages specify a specific release (or sometimes just a specific commit), ideally along with checksums for that snapshot of the download.

Currenly the only other difference between the two PKGBUILDs is that mine installs the Rust toolchain locally, in accordance with Arch's packaging guidelines (PKGBUILDs aren't supposed to modify files outside of the local `pkg` and `src` directories), while the other package doesn't. I decided to leave it that way for correctness, but I left instructions for how to change this in the PKGBUILD.

In the future, the [tiny-irc-client](https://aur.archlinux.org/packages/tiny-irc-client/) package will look very different once we start releasing binaries for tiny, as it will no longer use Rust or compile anything. It will be the contender to eventually move to the official repositories, while [tiny-irc-client-git](https://aur.archlinux.org/packages/tiny-irc-client-git/) will stay in the AUR and continue to compile tiny.

TL;DR the -git AUR package should work fine and currently both AUR packages are basically the same.
